index.js

```diff
@@ -29,13 +29,18 @@ app.use(expressValidator());
 app.use(methodOverride('_method'));
 app.use(express.static(path.join(__dirname, 'public')));
 
-// // Controllers
-// const phoneController = require('./controllers/phone');
-
-// app.post('/new_phone_call', phoneController.newCall);
-// app.get('/new_phone_call', phoneController.newCallTestGet);
-// app.post('/redir_call_for_zip', phoneController.redirectCall);
-// app.get('/redir_call_for_zip', phoneController.redirectCallTest);
+// ngrok For Testing
+var ngrok = require('ngrok');
+ngrok.authtoken(process.env.NGROK_AUTHTOKEN, function(err, token) {});
+ngrok.connect(function (err, url) {});
+
+// Controllers
+var smsController = require('./controllers/sms.controller');
+
+// SMS Routes
+app.route('/sms')
+  .get(smsController.index)
+  .post(smsController.receiveWoke);
```

Comment:
we're not using semi-colons in this repo. (i need to setup a lint:fix cmd so it's easy to keep this consistent.) Can you remove semicolons here and merge the upstream development branch and resolve and conflicts? Let me know if you have any questions about the upstream changes.